app/org/sagebionetworks/bridge/play/controllers/ScheduleController.java

```diff
@@ -40,4 +40,8 @@ public Result getSchedules() throws Exception {
         return okResult(schedules);
     }
     
+    @Deprecated
+    public Result getSchedulesV1() throws Exception {
+        return getSchedulesV1();
+    }
```

Comment:
Luckily the new PR renders most comments about these multiple methods moot. Regarding the tests, yes, we should merge this and see integration tests in development before merging the SDK's update to the v3 URLs.